CarInspection/src/WebCarInspection/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,52 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - Car inspectios</title>
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/css/bootstrap-datetimepicker.min.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
+    <script src="~/lib/jquery/dist/jquery.min.js"></script>
```

Comment:
It's bad practice.
All JS files should be in the script sections at the bottom of the document because in the current case your site will freeze until these JS scripts loading.
Fix it.

Reply:
some scripts must be loaded before the page loads. For example, scripts for validation should be loaded first (if not, model validation will not output errors)

Reply:
I don't see any validation scripts here)
And it still doesn't mean that you should use bad practice.
Please, fix this issue.